### PR TITLE
feat: allow to configure keymanager token file path

### PR DESCRIPTION
**Pull request by nflaig**
**Motivation**

- https://github.com/ethereum/keymanager-APIs/pull/74

**Description**

Allow to configure keymanager token file path via `--keymanager.tokenFile /path/to/file` CLI flag

### CODE REVIEW

File: packages/cli/src/cmds/validator/options.ts [hunk 1]
```diff
@@ -93,12 +94,18 @@ export const keymanagerOptions: CliCommandOptions<KeymanagerArgs> = {
     default: false,
     group: "keymanager",
   },
-  "keymanager.authEnabled": {
+  "keymanager.auth": {
```
Comment (nflaig) [inline packages/cli/src/cmds/validator/options.ts:97]: Updated the flag to enable authentication to be more inline with other existing boolean flags

Comment (jeluard) [reply packages/cli/src/cmds/validator/options.ts:97]: Probably worth documenting in next release, so that people can migrate to the preferred syntax?

**Review by jeluard — DISMISSED** [review]

**Comment by jeluard** [comment]
@nflaig Is there a reason why this PR is in `Draft` mode?

File: packages/cli/src/cmds/validator/options.ts (hunk 1 above)
Comment (nflaig) [reply packages/cli/src/cmds/validator/options.ts:97]: There is no need to migrate, existing setups can keep previous value

**Comment by nflaig** [comment]
> @nflaig Is there a reason why this PR is in `Draft` mode?

Waiting for spec PR discussion to resolve, might wanna rename flag name

**Review by jeluard — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.18.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.18.0) :tada:

**Pull request merged**